packages/manager/test/__testutils__/readMSWFormData.ts

```diff
@@ -0,0 +1,39 @@
+import parseMultipartFormData from "parse-multipart-data";
+import { RestRequest } from "msw";
+
+type FormDataField =
+	| { data: Buffer; type: string; filename?: string }
+	| { data: string };
+
+export const readMSWFormData = async (
+	req: RestRequest,
+): Promise<Record<string, FormDataField>> => {
+	const boundary = parseMultipartFormData.getBoundary(
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		req.headers.get("Content-Type")!,
```

Comment:
Yes, we assert that the `Content-Type` header exists using `!`.

I changed the code so it checks if `Content-Type` exists. I think it's okay to ignore some ESLint rules in tests since it is not production code, but the fix is easy here so why not make it better. 🙂

```ts
const contentType = req.headers.get("Content-Type");

if (contentType == null) {
	throw new Error(
		"readMSWFormData can only be used on requests that have a `Content-Type` header. This request did not contain one.",
	);
}
```